src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 humanYear when animals age 0', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 0 humanYear when animals age <= 14', () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1 humanYear when animals age > 14 and <= 23', () => {
```

Comment:
The test description says 'should return 1 humanYear when animals age > 14 and <= 23', but the test checks for age 15 only. You should add more test cases to cover the range of ages from 15 to 23.